Resources/Private/Javascript/Components/CartItems.js

```diff
@@ -37,6 +37,12 @@ const CartItems = ({ emptyMessage = 'Your shopping cart is empty' }) => {
               <div className='c-cart-items__quantity'>{product.quantity}</div>
               <div className='c-cart-items__unit-price'>{formatCurrency(product.price.unitPrice)}</div>
               <div className='c-cart-items__total-price'>{formatCurrency(product.price.totalPrice)}</div>
+              <select value={product.quantity} onChange={(e) => apiClient.updateLineItem(product.id, { quantity: parseInt(e.target.value) })}>
+                {[...Array(product.quantityInformation.maxPurchase - product.quantityInformation.minPurchase + 1)].map((x, i) =>
```

Comment:
Would be best to move the whole product to a separate component and move all data transformation out of the rendering nad it is always better to use some real names instead of x and i

Reply:
Done